tools/Makefile.unix

```diff
@@ -505,7 +505,7 @@ gconfig: apps_preconfig
 
 savedefconfig: apps_preconfig
 	$(Q) APPSDIR=${CONFIG_APPS_DIR} EXTERNALDIR=$(EXTERNALDIR) kconfig-conf --savedefconfig defconfig.tmp Kconfig
-	$(Q) sed -i -e "/CONFIG_APPS_DIR=/d" defconfig.tmp
```

Comment:
Can we just use kconfig-tweak for the config patches?

Reply:
done